src/main/java/com/list/todo/entity/TaggedTask.java

```diff
@@ -0,0 +1,27 @@
+package com.list.todo.entity;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Builder
+public class TaggedTask extends BaseEntity {
```

Comment:
Ill-fitted naming. TaggedTask  class implies that there are two types of tasks: TaggedTask and just Task (presumably not tagged), whereas in reality we have only one type of task: **Task**, and TaggedTask is only mapping